editor/src/components/inspector/inspector-common.ts

```diff
@@ -272,7 +272,12 @@ export const hugContentsApplicableForContainer = (
           MetadataUtils.isPositionAbsolute(element)
         ),
     ).length > 0
+
+  const isElementGrid = MetadataUtils.isGridLayoutedContainer(
```

Comment:
I would rename this to `isGrid` so it does not stutter with `isGridElement` & co